### ISSUE TITLE
One question

### ISSUE THREAD

**Issue by BX-xb** [KevinMusgrave/pytorch-metric-learning#485]
Hello,when I use TripletMarginLoss, it seems to use all possible triplets to compute the loss. How I can use the  anchor I want, not all anchors?

**Comment by KevinMusgrave** [comment]
Like this:

```python
import numpy as np

triplet_idx = np.array([[0, 5, 7], [4, 9, 10]])
a, p, n = np.hsplit(triplet_idx, 3)

loss = loss_fn(embeddings, labels, (a, p, n))
```

**Comment by BX-xb** [comment]
Thank you for your reply!
